src/app/plan/components/Wizard/HooksFormContainer.tsx

```diff
@@ -0,0 +1,87 @@
+import { withFormik, FormikProps } from 'formik';
+import utils from '../../../common/duck/utils';
+import HooksFormComponent from './HooksFormComponent';
+
+const AddEditHooksFormContainer: any = withFormik({
```

Comment:
The `any` type annotation is probably not doing us any favors here. At the very least it should be `React.FunctionComponent<any>`, but ideally we'd have a full props interface here. Looks like we would just need to import `IOtherProps` from `HooksFormComponent`.

Also, we should pass generic types into `withFormik` like we do in `WizardContainer`, e.g. `withFormik<IOtherProps, IFormValues>(...)`. That'll typecheck the values we return in here.